dockerfile/README.md

```diff
@@ -0,0 +1,42 @@
+# Dockerfile for WSO2 Data Analytics Server #
+
+The Dockerfile defines the resources and instructions to build the Docker image for WSO2 Data Analytics Server 3.1.0.
+
+## Quickstart guide
+
+ Steps to build the WSO2 Data Analytics Server 3.1.0 Docker image and run in your local machine are as follows:
+ 
+ The local copy of the `Dockerfile` directory will be referred as, `DOCKERFILE_HOME`.
+ 
+ * Create a directory named `files` inside `DOCKERFILE_HOME`
+     - This will result in a directory structure `<DOCKERFILE_HOME>/files`.
+ 
+ * Add the JDK and WSO2 API Manager distributions
+     - Download JDK 1.8 (http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html) and copy it to `<DOCKERFILE_HOME>/files`.
+     - Download the WSO2 Data Analytics Server 3.1.0 distribution (https://wso2.com/analytics) and copy it to `<DOCKERFILE_HOME>/files`.
+ 
+ * Build the Docker image
+     - Navigate to `<DOCKERFILE_HOME>` directory.
+     - Execute the `docker build` command as shown below;
+         + `docker build -t wso2das:3.1.0 files/`
```

Comment:
Shouldn't this be pointing to the current folder? 
```docker build -t wso2das:3.1.0 .```

Reply:
It is possible to simply add all the software distributions (product pack and JDK) to the current directory, where the Dockerfile is resides.
But primarily for the purpose of clarity, all software distributions required during the Docker image build has been added to `files/` directory, which is expected to reside in the same directory as the Dockerfile. Necessary instructions has been given in the quick start guide.

Reply:
@imesh yes, I have corrected it.